packages/vizbuilder/src/index.js

```diff
@@ -137,6 +143,9 @@ Vizbuilder.propTypes = {
 
 Vizbuilder.defaultProps = {
   config: {},
+  formatting: {
+    Dollars: d => `$${formatAbbreviate(d * 1 || 0)}`
+  },
```

Comment:
If the user supplies their own formatters for other annotations, will that delete the preset here? If so, we have to implement this in a way so that if a user passed `formatting={{Time: d => "this is time"}}` it wouldn't delete the default `Dollars` formatter.